packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantDishes/RestaurantDish.vue

```diff
@@ -9,6 +9,7 @@
                 {{ name }}
             </span>
             <span
+                v-if="calories || portion"
```

Comment:
does this need a unit test?

Reply:
Probably not, as all the current unit tests are passing (which test to ensure portions and calories are displayed), and we'd simply test if a span does or does not exist

Reply:
ah I just noticed this span is empty, what is it for?

Reply:
ah yes - sorry I missed that